www/js/input-page-anon.js

```diff
@@ -0,0 +1,698 @@
+/*This file includes all logic for populating SURVEY table. It does not include any logic for survey sumbission */
+
+var DOMAIN = "http://develop-caterpillars.vipapps.unc.edu";
+
+var leafPhotoTaken = false;
+
+//Tracks whether a photo was taken for the arthropod submission
+//Reset after each arthropod is saved
+var arthropodPhotoTaken = false;
+
+//Track whether circle count for selected site has been retrieved.
+var circleCountRetrieved = false;
+
+var temperature;
+var time;
+var date;
+var dateTime;
+var siteID;
+var sitePassword;
+var surveyType;
+var circle;
+var survey;
+var plantSpecies;
+var leafCount;
+var herbivoryValue;
+
+var leafImageURI;
+var numberOfArthropodsToSubmit;
+var numberOfArthropodsSubmitted;
+//Tracks which screen is currently being displayed
+var onArthropodPage = false;
+
+var temperatures = {
+	"<30" : {min : - 10, max : 30},
+	"30-39": {min : 30, max : 39},
+	"40-49": {min : 40, max : 49},
+	"50-59": {min : 50, max : 59},
+	"60-69": {min : 60, max : 69},
+	"70-79": {min : 70, max : 79},
+	"80-89": {min : 80, max : 89},
+	"90-99": {min : 90, max : 99},
+	"100-109": {min : 100, max : 109 },
+	">109": {min : 110, max : 130}
+};
+
+//Dropdown plugin data for herbivory list
+var ddData = [
+	{
+		value: 0,
+		selected: false,
+		imageSrc: "pictures/none.png"
+	},
+	{
+
+		value: 1,
+		selected: false,
+		imageSrc: "pictures/trace.png"
+	},
+	{
+		value: 2,
+		selected: false,
+		imageSrc: "pictures/light.png"
+	},
+	{
+		value: 3,
+		selected: false,
+		imageSrc: "pictures/moderate.png"
+	},
+	{
+		value: 4,
+		selected: false,
+		imageSrc: "pictures/heavy.png"
+	}
+];
+var db;
+document.addEventListener("deviceready", onDeviceReady, false);
+//Return to start screen if android back button is pressed
+function onDeviceReady(){
+
+	db=window.sqlitePlugin.openDatabase(
+        {name: 'app.db', location: 'default'}, 
+        DBSuccessCB(), 
+        function(error){alert("Error Open Database:"+JSON.stringify(error));}
+    );
+    function DBSuccessCB(){
+        alert("DB open OK");
+            //retrive site data from server
+        
+        //retrive sites with permission
+        
+    };
+	//alert("begin wait");
+	setInterval(retrieveSiteList(),500);
+
+	document.addEventListener("backbutton", function(e){
+		e.preventDefault();
+		//If displaying arthropod screen, return to main select screen
+		if(onArthropodPage){
+			returnToMainSelectScreen();
+		}
+		//Otherwise ask if the user wants to exit the app
+		else{
+			navigator.notification.confirm(
+					'Do you want to quit?',
```

Comment:
This seems a lot weird to me, that user press back button to exit the app on input-page. I thought pressing the back button should only bring you back to homepage-anon?